### DISCUSSION TITLE
Getting an Error While trying to Generate a Html report.

### DISCUSSION THREAD

**Discussion by Ajay-Gaddam** [TroyWalshProf/SeleniumAxeDotnet#361] [General]
![image](https://github.com/TroyWalshProf/SeleniumAxeDotnet/assets/83690329/b91cbcff-ae6e-4f20-962a-c571410f611f)

**Comment by TroyWalshProf** [comment]
> ![image](https://github.com/TroyWalshProf/SeleniumAxeDotnet/assets/83690329/b91cbcff-ae6e-4f20-962a-c571410f611f)
> 

Roughly 2 years ago, we switched over to Selenium 4.  There was a compatibility break between Selenium 3 and 4.

We strongly recommend updating to Selenium 4.  If you insist on staying with Selenium 3 you will need to use an older version, such as https://github.com/TroyWalshProf/SeleniumAxeDotnet/releases/tag/v3.0.1
